### PR TITLE
fix: search projects title

### LINKED ISSUE #4122: Searched projects name not visible properly
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

The name of the project is not displayed properly.

### Steps To Reproduce

1. Search for projects with long titles.

### Expected Behavior

The complete name of the project should be displayed.

### Screenshots

Before
image

After
image

### Device Information [optional]

```markdown
- OS:
- Browser:
- version:
```

### Additional context

_No response_

### Are you working on this issue?

Yes

### PR DESCRIPTION
**Pull request by disha1202**
Fixes #4122 

#### Describe the changes you have made in this PR -

1. Use ellipse overflow
2. Show full name in tooltip

### Screenshots of the changes (If any) -

image

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by disha1202** [comment]
@vedant-jain03 @tanmoysrt 
Please take a look. 
Also I haven't added screenshots as I am unable to test it on local.

**Comment by vedant-jain03** [comment]
> @vedant-jain03 @tanmoysrt Please take a look. Also I haven't added screenshots as I am unable to test it on local.

Thanks so how have you tested the changes @disha1202 ?

**Comment by tanmoysrt** [comment]
> > > @vedant-jain03 @tanmoysrt Please take a look. Also I haven't added screenshots as I am unable to test it on local.
> > 
> > 
> > Thanks so how have you tested the changes @disha1202 ?
> 
> I haven't tested it on local though I took reference from the [existing code](https://github.com/CircuitVerse/CircuitVerse/blob/e73103de5041d5f94fe38833233c59deeab6e198/app/assets/stylesheets/users.scss#L286) which working perfectly fine.
> 
> Hoping this would work as well :)
> 
> PS: If there is any way to test it locally please let me know.
Checkout the [Setup Documentation](https://github.com/CircuitVerse/CircuitVerse/blob/master/SETUP.md)
Try out `Github Codespaces` or `GitPod` to setup and run quickly.

**Comment by disha1202** [comment]
> > > > @vedant-jain03 @tanmoysrt Please take a look. Also I haven't added screenshots as I am unable to test it on local.
> > > 
> > > 
> > > Thanks so how have you tested the changes @disha1202 ?
> > 
> > 
> > I haven't tested it on local though I took reference from the [existing code](https://github.com/CircuitVerse/CircuitVerse/blob/e73103de5041d5f94fe38833233c59deeab6e198/app/assets/stylesheets/users.scss#L286) which working perfectly fine.
> > Hoping this would work as well :)
> > PS: If there is any way to test it locally please let me know.
> 
> Checkout the [Setup Documentation](https://github.com/CircuitVerse/CircuitVerse/blob/master/SETUP.md#installation) Try out `Github Codespaces` or `GitPod` to setup and run quickly.

Thanks @tanmoysrt
Updated the code and also added screenshot.

File: app/assets/stylesheets/search.scss [hunk 1]
```diff
@@ -76,12 +76,18 @@
   }
 }
 
-.search-project-name {
+.search-project-name h3 {
```
Comment (vedant-jain03) [inline app/assets/stylesheets/search.scss:79]: Try this:
```
.search-project-name {
  height: 40px;
  text-align: left;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
}

.search-project-name h3 {
  display: inline;
}
```

Why?
- We make overflow to parent of the text, in the case `h3` is block element, so we make it inline to ensure that ellipse works fine for that
- With the initial code, not able to understand why you made it `70%` but with the initial code, even the name is `Public` it is getting ellipsed which ideally should not happen

Screenshot 2023-10-09 at 9 38 49 PM

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]
So for the name "Public", the text is ellipsed which ideally should not happen

Screenshot 2023-10-09 at 9 38 49 PM

Possible solution:
- Make parent

**Comment by disha1202** [comment]
Hi @vedant-jain03 
Thanks for reviewing, I have updated the PR as per your suggestions. 
Please take a look.

**Review by vedant-jain03 — APPROVED** [review]

**Pull request merged**